essence-of-live-coding/test/Cell/Util.hs

```diff
@@ -192,4 +212,98 @@ test =
           }
     ]
 
+type TestTraversables' = TestTraversables '[Maybe, [], V.Vector 10, Seq, Map Int]
```

Comment:
Can we rename:

* `TestTraversables' -> TestTraversables`
* `TestTraversables -> Traversables`